release.yaml

```diff
@@ -0,0 +1,14 @@
+kuadrant:
+  release: "1.0.1"
```

Comment:
Once this goes to main branch, all the versions will be `0.0.0` or something else was decided?

Reply:
Nothing has being decided yet.

Reply:
In that case I'm in favour of `main` --> `0.0.0` as it's reflected currently on our manifests (OLM, Helm)